spotbugs/src/xsl/plain.xsl

```diff
@@ -205,7 +205,7 @@
 				<xsl:apply-templates select="$warningSet">
 					<xsl:sort select="@priority"/>
 					<xsl:sort select="@abbrev"/>
-					<xsl:sort select="Class/@classname"/>
+					<xsl:sort select="Class[1]/@classname"/>
```

Comment:
Does this still work if there is only one bug instance?

Reply:
According to W3C, `[1]` is the first element, so it should work.

* https://www.w3.org/TR/xslt20/
* https://www.w3schools.com/xml/xpath_syntax.asp

I'll check it later.